docs/_panels/attracted-contributors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Attracted Contributors
+description: attraction of contributors for Git and GitHub.
+author: Bitergia
+screenshot: sigils/attracted-contributors.png
+created_at: 2023-01-09
+grimoirelab_version: 0.2.2
+layout: panel
+---
+
+This panel shows the evolution of attracted contributors in the last ten years and the
+ones who recently joined. This analysis uses each person's first contribution date to
+estimate when they enter the project.
+
+This panel is intended to be used with the whole history of the community activated and
+not with some pre-established activity timeframe as in other panels (for instance,
+2 years). This is done this way as the information pre-calculated and displayed focuses
+on the first contribution of each new member.
+
+The drop-down menu on the top left side allows you to switch from Git to GitHub data sets,
+as a unified view is unavailable. Next to it is a small table with a summary of the number
+of total contributors that joined the project and the number of total organizations those
+newcomers belong to. On the left side, we can find a table with a list of newcomers, with
+a link to the SortinHat profile, the name of the organization they belong to, and the date
```

Comment:
Please, fix the typo `SortinHat` -> `SortingHat`.

Reply:
Fixed.